cmd/derper/derper.go

```diff
@@ -141,6 +150,15 @@ func writeNewConfig() config {
 	return cfg
 }
 
+func checkMeshKey(k string) (key string, err error) {
+	key = strings.TrimSpace(string(k))
+	if !validMeshKey.MatchString(key) {
+		err = fmt.Errorf("key in %s must contain 64+ hex digits", k)
+	}
+
+	return
+}
```

Comment:
In light of https://go.dev/wiki/CodeReviewComments#naked-returns and https://go.dev/wiki/CodeReviewComments#named-result-parameters, I'd recommend simplifying this:

```suggestion
func checkMeshKey(key string) (string, error) {
	key = strings.TrimSpace(key)
	if !validMeshKey.MatchString(key) {
		return "", fmt.Errorf("key in %q must contain 64+ hex digits", key)
	}
    return key, nil
}
```